Build/Common.proj

```diff
@@ -73,7 +72,7 @@
   </PropertyGroup>
 
   <!-- References -->
-  <Import Project="$(DlrBuildDir)\$(ReferencedPlatform).props" />
```

Comment:
Is TargetFramework always set? If not, don't we need this so that it pulls in when it's not set?